BasicsCodelab/app/src/main/java/com/codelab/basics/MainActivity.kt

```diff
@@ -57,16 +63,11 @@ fun MyApp(content: @Composable () -> Unit) {
 }
 
 @Composable
-fun MyScreenContent(names: List<String> = listOf("Android", "there")) {
+fun MyScreenContent(names: List<String> = List(1000) { "Hello Android #$it" }) {
     val counterState = remember { mutableStateOf(0) }
 
     Column(modifier = Modifier.fillMaxHeight()) {
-        Column(modifier = Modifier.weight(1f)) {
-            for (name in names) {
-                Greeting(name = name)
-                Divider(color = Color.Black)
-            }
-        }
+        NameList(modifier = Modifier.weight(1f), names = names)
```

Comment:
Since you reordered the function parameters below, this can be called without naming them

```suggestion
        NameList(names, Modifier.weight(1f))
```